### PR TITLE
Added solution for codility exercise

### PR DESCRIPTION
**Pull request by EtriKodoku**
Added solution for codility task MissingInteger

### CODE REVIEW

**Review by WebUraXalys — APPROVED** [review]

File: exercise.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+# Test data [1,2, 4], [-1,-3], [1,2,3]
+def solution(A):
+    a_list = sorted(A)
```
Comment (yurak) [inline exercise.py:4]: `sorted_list`

File: exercise.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+    if 1 not in a_list or a_list[-1] < 0:
+        return 1
+
+    for index in range(len(A)):
+        if index + 1 == len(A):
+            return a_list[index] + 1
+        elif a_list[index + 1] - a_list[index] > 1 and a_list[index] + 1 > 0:
+            return a_list[index] + 1
```
Comment (yurak) [inline exercise.py:13]: what is the difference between 11 and 13?

Comment (yurak) [reply exercise.py:13]:
```
if (index + 1 == len(A)) or  (a_list[index + 1] - a_list[index] > 1 and a_list[index] + 1 > 0):
  return a_list[index] + 1
```

Comment (yurak) [reply exercise.py:13]: a_list[index] + 1   - next or next_element

File: exercise.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+# Test data [1,2, 4], [-1,-3], [1,2,3]
+def solution(A):
+    a_list = sorted(A)
+
+    if 1 not in a_list or a_list[-1] < 0:
```
Comment (yurak) [inline exercise.py:6]: `if 1 not in a_list ` condition is enough

Comment (yurak) [reply exercise.py:6]: `return 1 if 1 not in a_list`

File: exercise.py (hunk 2 above)
Comment (yurak) [reply exercise.py:13]: a_list[index + 1] - a_list[index] > 1 - elements_difference
a_list[index] + 1 > 0 - next_is_positive?

File: exercise.py (hunk 1 above)
Comment (yurak) [reply exercise.py:4]: solution([1,2, 4]) ==3 
solution( [-1,-3]) ==1
solution([1,2,3]) == 4

Comment (yurak) [reply exercise.py:4]: def print_test_solution(solution):
    print(solution)

**Pull request closed without merging**